src/commands/rpc.ts

```diff
@@ -0,0 +1,125 @@
+import chalk from "chalk";
+import { CommandModule } from "yargs";
+
+import { getAndVerifyCredentials } from "../utils/credentials";
+import {
+  downloadGithubFolder,
+  saveEnvVariablesToFile,
+} from "../utils/fileSave";
+import { createPlaygroundQuery } from "../utils/playgroundQuery";
+import { ResourceByEnv, getResourceByName } from "../utils/resources";
+
+const inquirer = require("inquirer");
+
+const command = "rpc";
+const describe = "Interface with Retool RPC.";
+const builder = {};
+const handler = async function (argv: any) {
+  const credentials = getAndVerifyCredentials();
+  const origin = credentials.origin;
+
+  const { resourceName } = (await inquirer.prompt([
+    {
+      name: "resourceName",
+      message: "What is your RPC resource ID?",
+      type: "input",
+    },
+  ])) as { resourceName: string };
+
+  let resourceId: number;
+
+  if (resourceName === "") {
+    // TODO: Potentially add logic to create a resource here.
+    console.log("Please enter a valid RPC resource ID");
+    return;
+  } else {
+    const resourceByEnv = await getResourceByName(resourceName, credentials);
+    validateResourceByEnv(resourceByEnv);
+
+    resourceId = resourceByEnv["production"].id;
+  }
+
+  const { rpcAccessToken } = (await inquirer.prompt([
+    {
+      name: "rpcAccessToken",
+      message: `What is your RPC access token? You can add a new one here: ${origin}/settings/api`,
+      type: "password",
+    },
+  ])) as { rpcAccessToken: string };
+  if (rpcAccessToken === "") {
+    console.log("Please enter a valid RPC access token");
+    return;
+  }
+
+  const { appType } = (await inquirer.prompt([
+    {
+      name: "appType",
+      message: "Which of the following app templates would you like to use?",
+      type: "list",
+      choices: [
+        {
+          name: "Hello World!",
+          value: "hello_world",
+        },
+      ],
+    },
+  ])) as { appType: string };
+
+  const { destinationPath } = (await inquirer.prompt([
+    {
+      name: "destinationPath",
+      message: "Where would you like to create this app?",
+      type: "input",
+      default: "./" + "retool_rpc" + "_" + appType,
+    },
+  ])) as { destinationPath: string };
+
+  const queryResult = await createPlaygroundQuery(resourceId, credentials);
+
+  console.log("Creating app...");
```

Comment:
Nit: Move above `createPlaygroundQuery` ?

Nit: the logging pattern I've been following for potentially long, async tasks is to 

```
const spinner = ora("Creating app").start();

// async stuff

spinner.stop()
```

Reply:
good catch! used the spinner and started it before creating the playground query :)